demos/serverless/README.md

```diff
@@ -28,6 +28,16 @@ npm install
 npm run deploy -- -r us-east-1 -b <my-bucket> -s <my-stack-name> -a meeting
 ```
 
+#### ChimeSDKMeetings vs Chime
```

Comment:
Suggest to add namespace in front after Chime:
"ChimeSDKMeetings vs Chime namespace"

Reply:
fixed